include/mata/nfa.hh

```diff
@@ -514,6 +494,15 @@ public:
         return this->has_trans({src, symb, tgt});
     } // }}}
 
+    /**
+     * Get transitions from @p state_from using transition @p symbol.
+     * @param state_from State from which to get transitions.
+     * @param symbol Transition symbol to look for in transitions.
+     * @return Iterator for @c TransitionList; Allows us to compare to TransitionsList::end() to see whether the specified
+     *  transitions exist. If they do, we can access the transitions with a dereference operator (*iter).
+     */
+    TransitionList::const_iterator get_transitions_from_state_with(State state_from, Symbol symbol) const;
```

Comment:
I don't like `with` in name of function. What about `get_transitions_from_state_symbol`? It is not also nice but I don't have any better ideas.

Reply:
Then, what are you saying is that `get_transitions_from_state_with_symbol` is not an option, either?

Reply:
If there is not a clash with some other function, how about just simply `get_transitions_from`? I'd say the `State` and `Symbol` as parameters is implied here. Otherwise this would be called `get_transition_map_from`, e.g.

Reply:
I have no objections to `get_transitions_from()`. That is actually the name I have first used. However, I decided to change it to the current one because I thought it better represents the function. If the simpler name is clearer for you, I am OK with that.

Reply:
yes, use `get_transitions_from`.

Reply:
Done. I will rename functions using `get_transitions_from_state...()`, too. This way, we will have `get_transitions_from(from_state)` and overloaded `get_transitions_from(from_state, symbol)`. The user can then choose whether they want all transitions from a state, or just those with a specified symbol.